chainer/functions/normalization/batch_normalization.py

```diff
@@ -449,6 +472,22 @@ def _zero_if_none(xp, x, shape, dtype):
     return x
 
 
+def _get_dtype_of_tensor_descriptor(desc):
+    cudnn_dtype, _, _, _, _, _, _, _, _ = libcudnn.getTensor4dDescriptor(
```

Comment:
Use `cudnn_dtype, *_ = ...`, which is easier to read.

Reply:
Thanks, will update in that way.

Reply:
Seems this way of writing is not OK with python 2.7?

Reply:
Oh, sorry, I didn't notice that. Then it's ok to leave the current code as is.